ch03.asciidoc

```diff
@@ -386,18 +386,42 @@ This includes which public payment channels are connecting nodes, which capacity
+The routing scheme is called the "SPHINX mixformat" and will be explained in detail in a later chapter.
+Note that Lightning Onion routing via "SPHINX mixformat" is only similar to the TOR network routing in conecpt, but as a protocol as well as in implementation it is definitely different from the TOR network.
+For now we want to focus on its properties for the transport of payments.
+A payment package used for routing is called an "onion".
+footnote:[The term "onion" was originally used by the TOR project.
+The acronym TOR stands for "The Onion Router".
+Moreover, the Tor network is also called the Onion network. 
+The Tor project uses an onion as its logo and the top level domain name used by TOR services on the internet is ".onion".]
+As a simplified physical analogy imagine the payment package as a sealed box with a onion inside. 
+On its route from payer to payee the sealed box is handed from node to node. 
+Sealed just means that the box is encrypted and only the receiving node can open it.
+As will be explained, each node opens the box (using a cryptographic key), takes out the onion, and peals off the outermost layer of this onion.
+The receiving node can only read this outermost layer, no further layers, achieved through cryptography.
+This outermost layer tells the node if the box needs to be routed further and to which node. 
+The node than places the remaining, but one layer smaller, onion into the box and sends the box to the next node on the routing path.
+This continues until the receiving node is the payee and the routing stops and the payee can read the information encrypted in the innermost layer of the onion.
+Why use a box? 
+The box is used in this analogy to express that the size of the "package" sent along the route is always the same. 
+The first recipient receives a box of the same size as the last recipient, the payee.
+This is done to create privacy. 
+If not done, an observer could analyze the size of the communication and deduce how many hops there are on the route. 
+If the size does not change there is no way to tell if this is the first or the last hop in the route.
+The box in our analogy is what technically is called an "onion" in the Lightning network.
+
+1. The most important property is that a routing node can only see on which channel it received an onion and on which channel to set up an HTLCs and thus to which peer to forward the onion. This means that no routing node can know who initiated the payment and to whom the payment is destined. The exception is the payee. The payee knows that it is the final destination. 
+This achieves privacy.
+2. The onions are small enough to fit into a single TCP/IP package and actually even a link layer frame. This will make traffic analysis of the payments almost impossible, increasing privacy further. 
```

Comment:
I know this was left over, but this is a pretty strong claim that we need to modify later on. Onion routing alone isn't enough to thwart traffic analysis.

Reply:
I adjusted the expectations by modifying 3 sentences in 3 bullet points.